core-services/prow/02_config/openshift-assisted/assisted-installer-deployment/_prowconfig.yaml

```diff
@@ -0,0 +1,26 @@
+branch-protection:
+  orgs:
+    openshift-assisted:
+      repos:
+        assisted-installer-deployment:
+          branches:
+            master:
+              protect: true
+          protect: false
+tide:
+  merge_method:
+    openshift-assisted/assisted-installer-deployment: squash
+  queries:
```

Comment:
you should include or exclude in which branches you need this configuration to be applied.

Reply:
so changed to have those validations on all branches
is that right?

Reply:
correction: It shouldn't be mandatory though. Lets merge this.